adding-new-packages.md

```diff
@@ -1,11 +1,17 @@
-# Adding new packages to the Flatcar Linux OS image
+# Proposing new packages for inclusion into Flatcar Container Linux
 
-Flatcar’s mission statement:
+Flatcar Container Linux is modern Linux distribution for running container workloads. To stay modern, the packages included need to be kept up-to-date, and sometimes new packages introduced. This documents explains the process for the latter.
+
+## Project definition
+When proposing new packages for inclusion into Flatcar Container Linux, it's important to keep in mind how the project defines itself:  
 _Flatcar Container Linux is a fully open source, minimal-footprint, secure by default and always up-to-date Linux distribution for running containers at scale._
 
-The tools and applications included in the Flatcar Container Linux OS image are kept to a minimum, to reduce both image size as well as attack surface. Package addition requests are vetted accordingly, and are required to strongly contribute to the other principles in order to be accepted:
-- Secure by default: new package increases security. At a minimum the new package must not increase Flatcar’s attack surface.
-- Always up-to-date: new package improves Flatcar’s update mechanisms.
-- Running containers: new package improves container experience on Flatcar. Includes, but is not limited to, new container runtimes that offer improvements over the ones included with Flatcar.
-- At scale: new package improves automation of or telemetry served by Flatcar and/or eases operational burden.
+## New package criteria
+As a minimum Linux distribution, the tools and applications included in Flatcar Container Linux are to be kept to a minimum. This is to reduce both the image size and attack surface. Package addition requests are evaluated with this in mind. Other criteria that are weighed are the following.
+- ***Secure by default***: Does the package increase the security of Flatcar?
+- ***Always up-to-date***: Is the package actively maintained?
+- ***Running containers***: Does the package make Flatcar more relevant for container environments?
+- ***At scale***: Does the package improve automation of or telemetry served by Flatcar and/or ease operational burden.
```

Comment:
This one is missing the question mark at the end. Personally, I think I would rather have sentences and not questions, but it should be either all questions or all sentences.

Reply:
I think questions are confusing too, but for now i've added the question mark for consistency